test/dns/dns_api_test.rb

```diff
@@ -149,10 +149,10 @@ def test_create_aaaa_record_is_shortened
   end
 
   def test_create_cname_record
-    post '/', :fqdn => 'test.com', :value => 'test1.com', :type => 'CNAME'
+    post '/', :fqdn => 'test.com', :value => 'alias.com', :type => 'CNAME'
```

Comment:
This is the main important thing about this PR.
It'll change the signature of cname creation API.
I don't know if we should point out deprecation warning or keep a backwords compatibility